_docs/how-to/staking.md

```diff
@@ -0,0 +1,98 @@
+
+|Parameter|Description|
+|---|---|
+|balance|Unchanged. Total amount of FIO Tokens (SUFs) in account. Please note that this value will now include tokens which may not spendable, because they are staked or locked.|
+|available|Amount of FIO Tokens (SUFs) which can be spent. _available_ = _balance_ - _staked_ - _locked_.|
+|staked|Amount of FIO Tokens (SUFs) which are staked.|
+|srps|Amount of staking reward points held by account. See below for _Computing staking rewards_.|
+|roe|Rate of exchange of SRPs to FIO when unstaking. See below for _Computing staking rewards_.|
+
+### Getting information about locked tokens
+Total amount of locked tokens in account can be computed by using the values returned by [/get_fio_balance](https://developers.fioprotocol.io/pages/api/fio-api/#post-/get_fio_balance) call:
+```
+locked_tokens = balance - staked - available
+```
+
+In addition, information on when locked tokens will become available is returned using [/get_locks](https://developers.fioprotocol.io/pages/api/fio-api/#post-/get_locks) endpoint. It returns:
+
+|Parameter|Description|
+|---|---|
+|lock_amount|Amount of FIO Tokens (SUFs) that were locked in account.|
+|remaining_lock_amount|Amount of FIO Tokens (SUFs) still locked.|
+|time_stamp|Epoch date when locking clock was started.|
+|payouts_performed|Number of unlocks that have already occurred.|
+|can_vote|0 - user cannot vote locked tokens; 1 - user can vote locked tokens (for tokens locked as a result of unstake, 1 will be returned.)|
+|unlock_periods|Multiple unlock periods remaining and depicted as SUF _amount_ of FIO Tokens unlocking _duration_ seconds after _time_stamp_.|
+
+Please note that when user unstakes tokens multiple times, there may be multiple lock periods returned (unstakes during the same 24-hour period will only have 1 unlock period). Therefore, a maximum of 7 unlock periods will be found in accounts which staked tokens. However, locking can also be used for other purposes than staking unlocking, which means that some users may have more than 7 unlock periods.
+
+### Computing staking rewards
+When user has staked tokens, it's possible to compute their currently accrued rewards by using the values returned by [/get_fio_balance](https://developers.fioprotocol.io/pages/api/fio-api/#post-/get_fio_balance) call:
+
+```
+amount of staking rewards accrued = (srps * roe) - staked
+```
+
+A prorated amount of staking rewards accrued can also be used to compute partial unstaking. For example if user is has 1000 tokens staked and plans to unstake 200 tokens, they will receive 20% (200/1000) of total amount of staking rewards accrued.
+
+Please note that amount of staking rewards accrued calculation is approximate and may be very slightly different due to rounding.
+
+## Resources
+Looking for a non-technical guide to staking. Checkout [FIO Staking blog post](https://medium.com/fio-blog/fio-token-staking-fio-improvement-proposal-fip-21-explained-e80a43bf3e83)
```

Comment:
For external links, I usually add a tag that launches the link in a separate tab so the user does not have to hit the back button to navigate back to the dev hub. It would look like:

[FIO Staking blog post](https://medium.com/fio-blog/fio-token-staking-fio-improvement-proposal-fip-21-explained-e80a43bf3e83){:target="_blank"}

Optional.